src/Views/Config.elm

```diff
@@ -11,11 +18,6 @@ email =
     "hello@contemplatingaction.org.uk"
 
 
-helpline : String
-helpline =
-    "0000000000"
-
-
-helplineDisplay : String
-helplineDisplay =
-    "0000" ++ " " ++ "000" ++ " " ++ "000"
+callToAction : CallToAction
```

Comment:
@katjam Maybe confusing to have this as only a case change away its type name, but OK.

Reply:
Maybe - I thought about calling it callToActionData or instance or config - but seemed odd/confuising. I hope it will be ok to get used to.